Underline Strikethrough.roboFontExt/lib/Underline StrikeThrough.py

```diff
@@ -310,7 +310,19 @@ def updateFontList(self):
                             dictionary.update({fontIdentifier: value})
                         else:
                             dictionary.update({fontIdentifier: None})
-                        
+                # Deal with underline position
+                if 'public.openTypePostUnderlinePosition' in font.lib.keys():
+                    self.underlinePosition.update{fontIdentifier: font.lib['public.openTypePostUnderlinePosition'}
+                # If font doesn't have that lib key set, but is using fontmake, use fontmake behavior
+                elif font.lib["com.robofont.fontCompilerTool"] == 'fontmake':
+                    self.underlinePosition.update{fontIdentifier: font.info.postscriptUnderlineThickness}
```

Comment:
Should this be:
`self.underlinePosition.update{fontIdentifier: font.info.postscriptUnderlinePosition}`